test/fixtures/named-views/nuxt.config.js

```diff
@@ -1,30 +1,32 @@
 export default {
   router: {
     extendRoutes(routes, resolve) {
-      const indexIndex = routes.findIndex(route => route.name === 'index')
-      let index = routes[indexIndex].children.findIndex(route => route.name === 'index-child-id')
-      routes[indexIndex].children[index] = {
-        ...routes[indexIndex].children[index],
+      const indexChildRoute = routes.find(route => route.name === 'index')
+        .children.find(route => route.name === 'index-child-id')
```

Comment:
```suggestion
        .children.find(route => route.name === 'index-child-id')
```

could we extract that into a new variable? :relaxed:

Reply:
I agree, done!